uint16 index
uint16 command
string name
string description